app/src/main/java/com/aragh/kotlin2/screen/useralbums/UserAlbumsContract.kt

```diff
@@ -0,0 +1,20 @@
+package com.aragh.kotlin2.screen.useralbums
+
+import com.aragh.kotlin2.data.Album
+
+
```

Comment:
Same suggestion as in 
https://github.com/karolwartak/kotlin-android-testapp/pull/1/files#diff-2674322db6a6791a2feb032f881c9aa3